.github/workflows/action.yml

```diff
@@ -0,0 +1,20 @@
+name: A New GitHub Project card linked to a GitHub issue/pr to the specified project column
```

Comment:
I would make this something shorter like "Automate Issue Addition to Projects"

Reply:
Yea, I have done now